scylla-server/src/db_handler.rs

```diff
@@ -71,14 +64,12 @@ impl LocLock {
 /// A few threads to manage the processing and inserting of special types,
 /// upserting of metadata for data, and batch uploading the database
 pub struct DbHandler {
-    /// The list of nodes seen by this instance, used for when to upsert
-    node_list: Vec<String>,
     /// The list of data types seen by this instance, used for when to upsert
     datatype_list: Vec<String>,
     /// The broadcast channel which provides serial datapoints for processing
     receiver: Receiver<ClientData>,
-    /// The database
-    db: Database,
+    /// The database pool handle
```

Comment:
what is a pool handle 💀

Reply:
The handle for the pool of connections to the db.  Since a db is a mutable object we pass around this reference to the pool of connections, and get a connection when we need to talk to the db.